about.md

```diff
@@ -0,0 +1,21 @@
+---
+title: About
+sidebar: About
+layout: default
+permalink: /about/
+---
+
+GAPID is a developer tool for recording and inspecting calls made by an application to the graphics driver.
+
+Once a capture of the target application has been made, GAPID lets you disconnect from the target and inspect all the graphics commands made by the application.
+
+GAPID is able to replay the command stream, letting you visualize the frame composition draw-call by draw-call, and inspect the driver state at any point in the stream. Replay also supports modifications, allowing you to tweak command parameters and shader source to instantly see what effect this would have on the frame.
+
+GAPID can also visualize the textures, shaders and draw call geometry used by the application.
+
+GAPID support the OpenGL ES and Vulkan graphics APIs.
+GAPID can trace either on Android, and Vulkan applications on Windows, MacOS and Linux.
```

Comment:
There is no desktop vulkan support for MacOS. If this was meant to say that GAPID can run on all platforms, it would be best to separate the two statements to make it more clear.